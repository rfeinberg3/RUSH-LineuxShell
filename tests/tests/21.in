ls files > /tmp/output11
cat /tmp/output11
rm -f /tmp/output11
exit
