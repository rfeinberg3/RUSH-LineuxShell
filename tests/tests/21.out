rush> rush> tmp1
tmp2
tmp3
rush> rush> 